src/xcode/ENA/ENA/Source/Services/DCCValidationService/DCCValidationService.swift

```diff
@@ -0,0 +1,66 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
+import HealthCertificateToolkit
+
+protocol DCCValidationProviding {
+	func availableCountries(
```

Comment:
```suggestion
	func onboardedCountries(
```

To match the `ONBOARDED_COUNTRIES` naming from the errors we could rename this